### ISSUE TITLE
Database isn't sync with mapping for Sylius 1.11

### ISSUE THREAD

**Issue by delyriand** [Sylius/Sylius#13838]
**Sylius version affected**: 1.11

**Description**  

After a new installation of Sylius 1.11, the database schema isn't in sync with the current mapping file. Because the length of column queue_name [has changed from 255 to 190](https://github.com/symfony/doctrine-messenger/blob/5.4/Transport/Connection.php#L437) and missing two indexes on queue_name and available_at columns. 
The [Sylius migration file](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Migrations/Version20220203115813.php) creates the table with the old length.

**Steps to reproduce**  

- Install a Sylius 1.11
- Run the command: `bin/console doctrine:schema:validate`

**Possible Solution**  
Create a new migration file with the new length and indexes, like:

```php
    public function up(Schema $schema): void
    {
        if ($schema->hasTable('messenger_messages')) {
            $this->addSql('ALTER TABLE messenger_messages CHANGE queue_name queue_name VARCHAR(190) NOT NULL');
            $this->addSql('CREATE INDEX IDX_75EA56E0FB7336F0 ON messenger_messages (queue_name)');
            $this->addSql('CREATE INDEX IDX_75EA56E0E3BD61CE ON messenger_messages (available_at)');
        }
    }
```

**Comment by Ferror** [comment]
@delyriand that's a nice catch. Wanna create MR/PR?

**Comment by delyriand** [comment]
i can create a PR with a migration like other migration in CoreBundle ;)

**Comment by coldic3** [comment]
Hi @delyriand, closing as it was already fixed in Sylius v1.11.5 :beers:
